### PR TITLE
Update deployment doc for disconnected cluster

### PR DESCRIPTION
**Pull request by sm43**
Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: docs/DEPLOYMENT.md [hunk 1]
```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
 ```
 Replace `<access-token>` with your JWT token. you must have `config-refresh` scope to call this API. 
 
+## Deploying on Disconnected Cluster
+
+- To deploy on a disconnected cluster you need to fork the [tektoncd/catalog][catalog] or you can use your own catalog but it must follow the [Catalog TEP][catalog-tep].
+- Fork the tektoncd/hub and update the [hub config][hub-config] file for your catalog details.
```
Comment (PuneetPunamiya) [inline docs/DEPLOYMENT.md:435]: Could you please add/explain more or add an example of block of what needs to be updated in the config file ?

File: docs/DEPLOYMENT.md [hunk 2]
```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
...
+- Mirror the Hub Images to the registry disconnected cluster has access to. You can use release images or build your own. you can use the latest release tag as the image tag.
```
Comment (PuneetPunamiya) [inline docs/DEPLOYMENT.md:436]: Could you please reframe this ??

Comment (PuneetPunamiya) [reply docs/DEPLOYMENT.md:436]: Would it be nice to point to the `Build Image` section of the deployment docs ??

File: docs/DEPLOYMENT.md [hunk 3]
```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
...
+    - List of images to be mirrored
+        - quay.io/tekton-hub/ui
```
Comment (PuneetPunamiya) [inline docs/DEPLOYMENT.md:438]:
```suggestion
        UI  - quay.io/tekton-hub/ui
```

File: docs/DEPLOYMENT.md [hunk 4]
```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
...
+        - quay.io/tekton-hub/api
+        - quay.io/tekton-hub/db-migration
+        - postgres:13@sha256:260a98d976574b439712c35914fdcb840755233f79f3e27ea632543f78b7a21e
+- Now, follow the steps from [Deploy Database Section](#deploy-database) above. Make sure you update the images in deployment files before applying.
```
Comment (piyush-garg) [inline docs/DEPLOYMENT.md:442]: Can you please mention the step of how to do miroring

File: docs/DEPLOYMENT.md (hunk 2 above)
Comment (piyush-garg) [inline docs/DEPLOYMENT.md:436]: Can you please mention the step of how to do mirroring

File: docs/DEPLOYMENT.md (hunk 4 above)
Comment (piyush-garg) [inline docs/DEPLOYMENT.md:442]: Can you please mention, how the URLs will look like after mirroring for images

File: docs/DEPLOYMENT.md [hunk 5]
```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
 ```
 Replace `<access-token>` with your JWT token. you must have `config-refresh` scope to call this API. 
 
+## Deploying on Disconnected Cluster
+
+- To deploy on a disconnected cluster you need to fork the [tektoncd/catalog][catalog] or you can use your own catalog but it must follow the [Catalog TEP][catalog-tep].
```
[the remark this answers is not in the document]
Comment (sm43) [reply docs/DEPLOYMENT.md:434]: can we? disconnected cluster. they need to have a catalog in their env? right?

**Pull request merged**